recipes/r-mipfp/meta.yaml

```
@@ -0,0 +1,75 @@
+{% set version = '3.2.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-mipfp
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/mipfp_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/mipfp/mipfp_{{ version }}.tar.gz
+  sha256: 46b1328795d6fea3b87ade5d432346824b8442124b9a75b96f45ee80defdaabf
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-rsolnp
+    - r-cmm
+    - r-numderiv
+  run:
+    - r-base
+    - r-rsolnp
+    - r-cmm
+    - r-numderiv
+
+test:
+  commands:
+    - $R -e "library('mipfp')"           # [not win]
+    - "\"%R%\" -e \"library('mipfp')\""  # [win]
+
+about:
+  home: https://github.com/jojo-/mipfp
+  license: GPL-2.0-or-later

```

From CRAN meta data `# License: GPL-2` converts to:
```suggestion
  license: GPL-2.0-only

```

---

Thank you. I have committed the suggestion and checks have passed.